contracts/erc20-bridge-sampler/test/erc20-bridge-sampler.ts

```diff
@@ -34,10 +34,6 @@ blockchainTests('erc20-bridge-sampler', env => {
     const MAKER_TOKEN = randomAddress();
     const TAKER_TOKEN = randomAddress();
     let devUtilsAddress: string;
-    const FAKE_BUY_OPTS = {
```

Comment:
We no longer pass these opts into the sampler contract. They're hardcoded instead.